src/components/App.js

```diff
@@ -10,6 +12,9 @@ import TeacherMode from './modes/teacher/TeacherMode';
 import Header from './layout/Header';
 import Loader from './common/Loader';
 
+// bind katex to the window object
```

Comment:
@ahgentil @pyphilia, this will not interfere with Katex on the page view, right? window is bound to the iframe. Just to confirm...

Reply:
From what I know it will only impact the iframe, but I might be wrong.